### PR TITLE
Corrected typo and added one image description

### PR DESCRIPTION
**Pull request by jacottemonroe**
@GreatEmerald @Timmarh

### CODE REVIEW

File: index.Rmd [hunk 1]
```diff
@@ -140,6 +140,8 @@ names(wagLandsat) <- c("band1", "band2", "band3", "band4", "band5", "band6", "ba
 plotRGB(wagLandsat, 5, 4, 3) 
 ```
 
+We have chosen to visualize the Landsat image as a false color composite, meaning that the chosen bands do not match the RGB channels. Indeed, we have plotted the near-infrared band as red, the red as green, and the green as blue.
```
Comment (jacottemonroe) [inline index.Rmd:143]: The color of the image could be a bit confusing to students without any explanation. I added one line, but I don't know if that complicates things more.

**Comment by GreatEmerald** [comment]
Thanks!

**Review by jacottemonroe — COMMENTED** [review]
Added a bit more clarification. Otherwise not much to add to this tutorial, it seems accessible for students.

File: index.Rmd (hunk 1 above)
Comment (GreatEmerald) [reply index.Rmd:143]: No, it's helpful, thanks!

**Pull request merged**